ONSdigital/response-operations-ui
# added JSON processor to logger

# Motivation and Context
Adding JSON formatting for Stackdriver, as described in card "S8 Add log configuration to our services - format for Stackdriver (8)"

# What has changed
Added JSON processor to structlog.wrap_logger()




# How to test?
pipenv run python run_tests.py
make test
Spin up a dev cluster with rops using the image "cloud-logging" and check to see that Stackdriver can see discrete events in the container log. 




# Links
https://trello.com/c/wDSxYgCQ




# Screenshots (if appropriate):
